man/systemd-system.conf.xml

```diff
@@ -337,6 +336,17 @@
         <literal>VAR3</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ManagerEnvironment=</varname></term>
+
+        <listitem><para>Sets environment variables just for the manager process itself. Takes the same
+        arguments as <varname>DefaultEnvironment=</varname>, see above.</para>
```

Comment:
Maybe also emphasize:

`Note that these variables are not inherited to processes spawned by the service manager, use DefaultEnvironment= for that`.

Also maybe say that they are merged with whatever the kernel passes in